velox/exec/tests/SpillerBenchmarkBase.cpp

```diff
@@ -40,6 +40,10 @@ DEFINE_string(
     spiller_benchmark_compression_kind,
     "none",
     "The compression kind to compress spill rows before write to disk");
+DEFINE_string(
+    spiller_benchmark_spiller_type,
+    "AGGREGATE_INPUT",
+    "The spiller type name, must be in all uppercase");
```

Comment:
nit: in followup you could consider to add to upper case. Thanks!

Reply:
Got it, added in SpillerAggregateBenchmarkTest

Reply:
The follow-up you said is to add a factory method to produce the required spiller instance based on this type flag and then we could only need one executable say `SpillerBenchmarkTest`?